cli/tsc/99_main_compiler.js

```diff
@@ -1152,12 +1178,6 @@ delete Object.prototype.__proto__;
     debug("serverInit()");
   }
 
-  function serverRestart() {
-    languageService = ts.createLanguageService(host, documentRegistry);
-    isNodeSourceFileCache.clear();
```

Comment:
Does this break anything?

I'm not sure if we cleared this cache for a specific reason or if it's just "be safe and clear the cache" legacy code.

Reply:
Isn't this one used by `Deno: Restart Language Server` task in `vscode_deno`?

Reply:
It shouldn't in practice, this checks if a path contains a `/node_modules/` component or something similar. It shouldn't change values as a result of caching or changing watched files.

Actually to be safe I'll delete it on config changes. Maybe it's affected for non-byonm npm resolvers.

Reply:
`Deno: Restart Language Server` restarts this whole process.